packages/cpmd/package.py

```diff
@@ -40,11 +42,21 @@ class Cpmd(MakefilePackage):
 #    patch('file://{0}/patch.to.4650'.format(basedir), sha256='70e752db0b454db39ebf703944f35b584cf5bafc9addf88d372c75d04ceadbf2', level=0, when='@4.3')
 
     def edit(self, spec, prefix):
+
         # patch configure file
-        cbase = 'LINUX-GFORTRAN'
+        if spec.satisfies('%gcc'):
+            cbase = 'LINUX-GFORTRAN'
+        elif spec.satisfies('%intel'):
+            cbase = 'LINUX-INTEL'
```

Comment:
```suggestion
        cbase = 'LINUX-GFORTRAN'
        if spec.satisfies('%intel'):
            cbase = 'LINUX-INTEL'
```
Other why it will fail with any other compiler
If it should fail the `conflicts()` lines are missing